### PR TITLE
csle tolerance tests

### PR DESCRIPTION
**Pull request by Yuhu-kth**

### CODE REVIEW

**Comment by Yuhu-kth** [comment]
In the file "csle/simulation-system/libs/csle-tolerance/tests/test_intrusion_response_cmdp_util.py", line #145-147# :
Screenshot 2024-05-03 at 19 25 06

normalizing_constant is overwritten by line 151. So it's a bit weird to me.

**Comment by Limmen** [comment]
> In the file "csle/simulation-system/libs/csle-tolerance/tests/test_intrusion_response_cmdp_util.py", line #145-#147 : Screenshot 2024-05-03 at 19 28 40
> 
> normalizing_constant is overwritten by line 151. So it's a bit weird to me.

Yes you are right, good catch @Yuhu-kth. You can remove the redundant lines, i.e, 145-147

**Review by Limmen — COMMENTED** [review]
Everything looks good so far. Lets continue with this PR until you completed all tests for csle-tolerance, then I will merge it and we can move on to the next task.

File: simulation-system/libs/csle-tolerance/src/csle_tolerance/envs/intrusion_recovery_pomdp_env.py [hunk 1]
```diff
@@ -146,6 +146,7 @@ def _info(self, info: Dict[str, Any]) -> Dict[str, Any]:
         upper_bound_return = 0
         s = self.trace.states[0]
         for i in range(len(self.trace.states)):
+            print(i)
```
Comment (Limmen) [inline simulation-system/libs/csle-tolerance/src/csle_tolerance/envs/intrusion_recovery_pomdp_env.py:149]: Remove print statement

File: simulation-system/libs/csle-tolerance/src/csle_tolerance/util/general_util.py [hunk 2]
```diff
@@ -73,6 +72,9 @@ def register_envs() -> None:
 
         :return: None
         """
+        #import logging
```
Comment (Limmen) [inline simulation-system/libs/csle-tolerance/src/csle_tolerance/util/general_util.py:75]: Remove comments

**Review by Limmen — CHANGES_REQUESTED** [review]
Looks good, small things to fix

**Pull request merged**